NEWS.md

```diff
@@ -64,6 +64,8 @@
 
 * `object_name_linter()` gains parameter `regexes` to allow custom naming conventions (#822, #1421, @AshesITR)
 
+* `literal_coercion_linter()` reports a replacement in the lint message (#1439, @MichaelChirico)
```

Comment:
It's not immediately clear to me what this means. Maybe "`literal_coercion_linter()` reports the equivalent literal as a replacement in the lint message"?

Reply:
Gave some concrete examples, PTAL